### DISCUSSION TITLE
Permission structure for Map, Logbook & History

### DISCUSSION THREAD

**Discussion by ceddy441** [home-assistant/frontend#7461] [Configuration]
The Request
Add a new "group" of users called "Super Users" which would sit between Users and Administrators. 
"Super Users" and "Administrators"  would have access to Map, Logbook History.  Whereas "Users" would not have access. 
This will add additional privacy options when using HA in kiosk mode. In future this role could be used to further lock down changes. 

The Alternatives
You can "hide" the items, however a clever or knowledgeable user could re-add them.

Additional Information 
None Provided

**Comment by KTibow** [comment]
What's so secretive about map, logbook, and history? I wouldn't hide anything in there. IMO instead of
change users and admins to
users, super users, and admins
it'd be better to
change users and admins to
kiosk users, users, and admins

**Reply by ceddy441** [reply]
You hit the nail on the head.... When in Kiosk mode at the front door.. I don't need non-family members who are wow'd by Home Assistant being nosey and reviewing logs and maps. 

Isn't that the point of having self-hosted platform, to manage your own security and data residency?

**Reply by KTibow** [reply]
Well the sidebar is hidden by default, if not auto-detected make sure to turn this on:
![image](https://user-images.githubusercontent.com/10727862/97240618-f6a83f80-17ab-11eb-84dc-d8b740312b46.png)
